ipatests/test_integration/util.py

```diff
@@ -86,5 +87,5 @@ def ldappasswd_user_change(user, oldpw, newpw, master):
     userdn = "uid={},{},{}".format(user, container_user, basedn)
 
     args = [paths.LDAPPASSWD, '-D', userdn, '-w', oldpw, '-a', oldpw,
-            '-s', newpw, '-x']
+            '-s', newpw, '-x', '-H', api.env.ldap_uri]
```

Comment:
@MartinBasti I agree with your concerns. I guess constructed LDAP URI from passed in master parameter sounds as a safer bet.